docs/statistics.md

```diff
@@ -1,269 +1,425 @@
 
-## pktRecv
+Same as `pktSentTotal`, but for a specified interval. Available for sender.
 
-Same as `pktRecvTotal`, but for a specified interval.
+### pktRecv
 
-## pktSndLoss
+Same as `pktRecvTotal`, but for a specified interval. Available for receiver.
 
-Same as `pktSndLossTotal`, but for a specified interval.
+### pktSndLoss
 
-## pktRcvLoss
+Same as `pktSndLossTotal`, but for a specified interval. Available for sender.
 
-Same as `pktRcvLossTotal`, but for a specified interval.
+### pktRcvLoss
 
-## pktRetrans
+Same as `pktRcvLossTotal`, but for a specified interval. Available for receiver.
 
-Same as `pktRetransTotal`, but for a specified interval.
+### pktRetrans
 
-## pktRcvRetrans
+Same as `pktRetransTotal`, but for a specified interval. Available for sender.
+
+### pktRcvRetrans
 
 Same as `pktRcvRetransTotal`, but for a specified interval.
 
-## pktSentACK
+TODO: 
+
+- There is no `pktRcvRetransTotal` stats.
+- Which side
+- include in table
+
+### pktSentACK
 
 Same as `pktSentACKTotal`, but for a specified interval.
 
-## pktRecvACK
+TODO: Update accordingly.
+
+### pktRecvACK
 
 Same as `pktRecvACKTotal`, but for a specified interval.
 
-## pktSentNAK
+TODO: Update accordingly.
+
+### pktSentNAK
 
 Same as `pktSentNAKTotal`, but for a specified interval.
 
-## pktRecvNAK
+TODO: Update accordingly.
+
+### pktRecvNAK
 
 Same as `pktRecvNAKTotal`, but for a specified interval.
 
-## pktSndFilterExtra
+TODO: Update accordingly.
 
-Same as `pktSndFilterExtraTotal`, but for a specified interval.
+### pktSndFilterExtra
+
+Same as `pktSndFilterExtraTotal`, but for a specified interval. Sender only.
 
 Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## pktRcvFilterExtra
+### pktRcvFilterExtra
 
-Same as `pktRcvFilterExtraTotal`, but for a specified interval.
+Same as `pktRcvFilterExtraTotal`, but for a specified interval. Receiver only.
 
-Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+Introduced in SRT v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## pktRcvFilterSupply
+### pktRcvFilterSupply
 
-Same as `pktRcvFilterSupplyTotal`, but for a specified interval.
+Same as `pktRcvFilterSupplyTotal`, but for a specified interval. Receiver only.
 
-Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+Introduced in SRT v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## pktRcvFilterLoss
+### pktRcvFilterLoss
 
-Same as `pktRcvFilterLossTotal`, but for a specified interval.
+Same as `pktRcvFilterLossTotal`, but for a specified interval. Receiver only.
 
-Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+Introduced in SRT v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## mbpsSendRate
+### mbpsSendRate
 
 Sending rate in Mbps. Sender side.
 
-## mbpsRecvRate
+TODO: How it is calculated?
```

Comment:
`mbpsSendRate = 8 * pktSend / count_us(now - lastSampleTime)` (symbolically. `pktSend` is set to 0 at the `lastSampleTime` time).